src/reformatters/common/update_progress_tracker.py

```diff
@@ -38,21 +22,28 @@ class UpdateProgressTracker:
 
     def __init__(
         self,
-        store: zarr.abc.store.Store,
         reformat_job_name: str,
         time_i_slice_start: int,
+        store_path: str,
     ):
         self.reformat_job_name = reformat_job_name
         self.time_i_slice_start = time_i_slice_start
         self.queue: queue.Queue[str] = queue.Queue()
-        self.fs, self.path = _get_fs_and_path(store)
+
+        self.fs, self.update_progress_dir = fsspec.core.url_to_fs(
+            store_path.replace(".zarr", "_update_progress")
+        )
+
+        if isinstance(self.fs, fsspec.implementations.local.LocalFileSystem):
+            self.fs.makedirs(self.update_progress_dir, exist_ok=True)
```

Comment:
this is the fix -- explicitly make directory if it's local, otherwise we should be able to write to the path on object storage.